internal/provider/encodingcom/encodingcom_test.go

```diff
@@ -1216,7 +1216,7 @@ func TestPresetToFormat(t *testing.T) {
 	}
 	var p encodingComProvider
 	for _, test := range tests {
-		resultingFormat := p.presetToFormat(test.givenPreset)
+		resultingFormat, _ := p.presetToFormat(test.givenPreset)
```

Comment:
Should the test fail if the error is not nil here?

Reply:
Sure, I can add it in. Unlikely we'll ever get an error here, but good to cover our bases just in case.